component/component/app.jsonnet

```diff
@@ -3,7 +3,23 @@ local inv = kap.inventory();
 local params = inv.parameters.appcat;
 local argocd = import 'lib/argocd.libjsonnet';
 
-local app = argocd.App('appcat', '');
+local app = argocd.App('appcat', '') + (
+  if params.services.vshn.e2eTests then {
+    spec+: {
+      ignoreDifferences+: [
+        {
+          group: '',
+          jsonPointers: [
+            '/data',
+          ],
+          kind: 'Secret',
+          name: 'github-ci-secret',
+          namespace: 'schedar-e2e',
```

Comment:
I would keep the project name instead of team name.

Reply:
I wouldn't - it's descriptive for all engineers right now, it's also easy way to contact with us.

Reply:
It's currently deployed as `schedar-e2e`, this adjustment makes that the jsonnet actually matches what's deployed.

But if you like the old name more, I can change it.

Reply:
> I wouldn't - it's descriptive for all engineers right now, it's also easy way to contact with us. Component can be owned by different teams, `schedar` is just one

True, but schedar owns appcat, so it should be the same either way.

But thinking a bit more about the naming, it might make sense to go with `appcat-e2e`. Otherwise, we're leaking some vshn internal naming to the product.